addons/account/static/src/components/action_menus/action_menus.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="account.ActionMenus" t-inherit="web.ActionMenus" t-inherit-mode="extension">
+        <xpath expr="//span[@id='action_menu_print_span']" position="replace">
+            <t t-if="props.resModel === 'account.move'">
+                <span class="o_dropdown_title">Download</span>
+            </t>
+            <t t-else="">
+                <span class="o_dropdown_title">Print</span>
```

Comment:
It's dangerous to override the existing `web.ActionMenus` with `extension`. What about making the label dynamic and change it directly js-side?